src/OrchardCore.Modules/OrchardCore.Media/wwwroot/Scripts/media.js

```diff
@@ -267,7 +273,7 @@ function initializeMediaApplication(displayMediaApplication, mediaApplicationUrl
               return;
             }
 
-            confirmDialog(_objectSpread(_objectSpread({}, $("#deleteFolder").data()), {}, {
+            confirmDialog(_objectSpread({}, $("#deleteFolder").data(), {
```

Comment:
Before it was the opposite ;) as i think after some updates a full gulp built was not done again

But now if you update node and npm, do npm install and then a gulp build the generated line is

   confirmDialog(_objectSpread(_objectSpread({}, $("#deleteFolder").data()), {}, {